Find a functionally equivalent code:android.content.res.TypedArray.getIndexCount:COMMENT
Method Modifier: public      
Comment:/**
 * Return the number of indices in the array that actually have data.
 */

@@ -1,3 +1,6 @@
 {
+    if (mRecycled) {
+        throw new RuntimeException("Cannot make calls to a recycled instance!");
+    }
     return mIndices[0];
 }

